charts/datadog/values.yaml

```
@@ -1305,6 +1305,10 @@ clusterAgent:
   #          username: datadog
   #          password: <YOUR_CHOSEN_PASSWORD>
 
+  ## clusterAgent.kubernetesApiserverCheck.disableUseComponentStatus -- Disable useComponentStatus for kubernetes_apiserver check
+  kubernetesApiserverCheck:
+    disableUseComponentStatus: # true
```

`# true` is causing this to get interpreted as a string when generating the docs:

```suggestion
    disableUseComponentStatus: false
```